package.json

```
@@ -7,16 +7,18 @@
   "engines": {
     "node": ">=12.0.0"
   },
+  "publishConfig": {
+    "access": "public"
```

This should list the registry as well. At least that's what we do everywhere else

```suggestion
    "access": "public",
    "registry": "https://registry.npmjs.org/"
```